include/z64thread.h

```diff
@@ -0,0 +1,24 @@
+#ifndef Z64_THREAD_H
+#define Z64_THREAD_H
```

Comment:
```suggestion
#ifndef Z64THREAD_H
#define Z64THREAD_H
```
Just from a quick peak, could you fix `Z64_GAME_H` and `Z64_ANIMATION_LEGACY_H`?